ramls/inventory-view-instance.json

```diff
@@ -1,17 +1,26 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "Instance with holdings and items",
+  "description": "Instance with holdings and items (there will be all instance properties, they just omitted, see instance.json)",
   "type": "object",
   "properties": {
-    "instanceId": {
-      "description": "Instance id",
-      "$ref": "uuid.json"
+    "id": {
+      "type": "string",
+      "description": "Instance id"
+    },
+    "source": {
+      "type": "string",
+      "description": "The metadata source and its format of the underlying record to the instance record. (e.g. FOLIO if it's a record created in Inventory;  MARC if it's a MARC record created in MARCcat or EPKB if it's a record coming from eHoldings)"
     },
-    "instance": {
-      "description": "Instance record",
-      "$ref": "instance.json"
+    "title": {
+      "type": "string",
+      "description": "The primary title (or label) associated with the resource"
+    },
+    "instanceTypeId": {
+      "type": "string",
+      "description": "UUID of the unique term for the resource type whether it's from the RDA content term list of locally defined",
+      "$ref": "uuid.json"
```

Comment:
These appear to be breaking compatibility changes, though the interface version has only been given a minor version change.

Have I misinterpreted these changes?

Reply:
Yes, it is a breaking change, however the API has been created few days ago and it should have the `0.1` version (but it has 1.0 because of a typo).

Should I change version to 0.2 instead?

Reply:
@bohdan-suprun Thank you for checking my understanding.

The accidental declaration of a public API is unfortunate and does make this a more challenging decision.

I shall leave that decision to the Falcon team. 

The original work was feature work merged after the API freeze for 2021 R1, as I imagine this work is intended to be too.

Reply:
I'd better keep the previous schema, then, thanks!

Reply:
@bohdan-suprun 

> I'd better keep the previous schema, then, thanks!

As I say, I suggest this be discussed by the Falcon team

Reply:
> I'd better keep the previous schema, then, thanks!

You mean the previous composite schema with `instanceid, instace, holdings, items`? I like that one too. Feels cleaner.